### DISCUSSION TITLE
why add s at the end of the word?

### DISCUSSION THREAD

**Discussion by automataIA** [adityatelange/hugo-PaperMod#1153] [Q&A] [answered]
Why the theme add an S at the finish of the word. I tried to find where the theme change word, but i didn't found nothing.
![immagine](https://user-images.githubusercontent.com/53933533/218072087-d53608b4-8c1c-4405-82ad-6743dad3f30a.png)

the code in the yml file is:
```
              - name: Progetti
                url: progetti
```
The directory name is progetti.

**Comment by hobbesjaap** [comment]
I find the theme does the same when you have /blog and the page is displayed at "Blogs". Would like some help with this too if possible please, thank you!

**Comment by jesse-wei** [comment, marked as answer]
In your config file, add `pluralizeListTitles: false`.

[Source](https://discourse.gohugo.io/t/how-can-i-stop-section-title-from-showing-up-as-plural/994)

**Reply by hobbesjaap** [reply]
Thank you!!

**Comment by RoneoOrg** [comment]
@automataIA Could you please mark this question as "Answered"?

**Reply by automataIA** [reply]
did it.
